README.md

```diff
@@ -14,4 +16,4 @@ Current build status of development branch: [![Build Status](https://ci.mysensor
 
 Current build status of master branch (nightly build of Arduino IDE): [![Build Status](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/master/badge/icon)](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/master/)
 
-Current build status of development branch (nightly build of Arduino IDE): [![Build Status](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/badge/icon)](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/)
\ No newline at end of file
+Current build status of development branch (nightly build of Arduino IDE): [![Build Status](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/badge/icon)](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/)
```

Comment:
What is the actual change here?

Reply:
Good question. I tried to find it when I made the commit, but was unable to find any difference. My best guess is that vi made some automatic white space change. https://github.com/mysensors/MySensors/pull/1254/files/b31df35392a8f2625c1820050e749eed11a1f27c?w=1 (option=ignore whitespace changes) says that the line in question was not changed.